app/src/main/res/values/strings.xml

```diff
@@ -111,4 +111,5 @@
     <string name="symbol_percent"> %</string>
     <string name="title_post_detail">%1$s Post</string>
     <string name="title_selected_image_deital">Image detail</string>
+    <string name="description_write_a_comment">Write a comment...</string>
```

Comment:
WARNING: [TypographyEllipsis] Replace "..." with ellipsis character (…, &#8230;) ?
 You can replace the string "..." with a dedicated ellipsis character, ellipsis character (…, &#8230;). This can help make the text more readable.